test/test_parser.rb

```diff
@@ -6278,4 +6278,41 @@ def test_bug_do_block_in_hash_brace
       %q{},
       SINCE_2_3)
   end
+
+  def test_lparenarg_after_lvar__since_25
```

Comment:
Please add a test that the behavior before ruby25 stays the same.

Reply:
First two tests check it. Old versions are `ALL_VERSIONS - SINCE_2_5`. Should I add constants like `BEFORE_2_5` ?

Reply:
Oh sorry, I missed that.

Reply:
So... How about adding `BEFORE_X_Y` constants?